src/main/java/com/openshift/restclient/authorization/TokenAuthorizationStrategy.java

```diff
@@ -38,5 +38,19 @@ public void accept(IAuthorizationStrategyVisitor visitor) {
 		visitor.visit(this);
 	}
 
-
+	@Override
+	public int hashCode() {
```

Comment:
Why is this so complicated and not just either the hashcode of the token or String.hashCode()